packages/dds/tree/src/simple-tree/mapNode.ts

```diff
@@ -128,7 +128,8 @@ export interface TreeMapNode<T extends ImplicitAllowedTypes = ImplicitAllowedTyp
 	): void;
 }
 
-const handler: ProxyHandler<TreeMapNode> = {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Probably worth a quick comment explaining why this is okay (so no one things they need to clean it up)

Reply:
```suggestion
// TreeMapNode is invariant over schema type, so for this handler to work with all schema, the only possible type for the schema is `any`.
// This is not ideal, but no alternatives are possible.
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```